packages/common/src/EIP712/ForwarderUtil.ts

```diff
@@ -1,5 +1,5 @@
 import { GsnDomainSeparatorType, GsnRequestType } from './TypedRequestData'
-import { IForwarderInstance } from '@opengsn/contracts/types/truffle-contracts'
+import { IForwarderInstance } from '@oasislabs/opengsn-contracts/types/truffle-contracts'
```

Comment:
I thought `@opengsn/contracts` should be resolved to `@oasislabs/opengsn-contracts` automatically.

Reply:
ideally, this change should not be neccessary.